packages/entity/src/EntityPrivacyPolicy.ts

```diff
@@ -11,7 +11,19 @@ import PrivacyPolicyRule, { RuleEvaluationResult } from './rules/PrivacyPolicyRu
 /**
  * Information about the reason this privacy policy is being evaluated.
  */
-export type EntityPrivacyPolicyEvaluationContext = {
+export type EntityPrivacyPolicyEvaluationContext<
+  TFields extends object,
+  TID extends NonNullable<TFields[TSelectedFields]>,
+  TViewerContext extends ViewerContext,
+  TEntity extends ReadonlyEntity<TFields, TID, TViewerContext, TSelectedFields>,
+  TSelectedFields extends keyof TFields = keyof TFields
+> = {
+  /**
+   * When this privacy policy is being evaluated as a result of an update, this will be populated with the value
+   * of the entity before the update. Note that this doesn't only apply to UPDATE authorization actions though:
+   * when an entity is updated it is re-loaded after the update completes.
```

Comment:
Would it be clearer to say:
```suggestion
   * when an entity is updated it is re-LOADed after the update completes.
```
to highlight why the LOAD is different from the UPDATE?